### PR TITLE
Add a flag to explicitly log query

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

**Review by beets — APPROVED** [review]
should we add this to magic-eye too?

File: server/configmodule.py [hunk 1]
```diff
@@ -40,6 +40,10 @@ class Config:
   API_PROJECT = ''
   # Set this to False if the deployment has frequently updated data.
   USE_MEMCACHE = True
+  # Whether to log the query (and make them avaiable in query history interface)
+  # Eanble this to "True" requires adding "bigtable/user" acccess for the
```
Comment (pradh) [inline server/configmodule.py:44]: nit: Enabling

Comment (shifucun) [reply server/configmodule.py]: Done

**Comment by shifucun** [comment]
> should we add this to magic-eye too?

For custom DC, will need to use the Bigtable in the custom project. I will do that in follow up changes.

**Pull request merged**